### PR TITLE
Feat(PP-65): updated how highscores are stored and displaying them on `account-main` page

### PR DESCRIPTION
**Pull request by DmitrijSidorkin**

### CODE REVIEW

**Review by elnino97 — COMMENTED** [review]
This empty div could be removed

File: views/account/account-main.ejs
Comment (elnino97) [inline views/account/account-main.ejs:1]: Empty div at line 17 could be removed

File: public/javascripts/account-main.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            htmlRows[i] += `<div class="grid-score">0</div>`;
+          } else {
+            htmlRows[i] += `<div class="grid-score">${
+              data.highscores[i + 1][data.lengthSettingsOptions[j]]
+            }</div>`;
+          }
+        }
+      }
+      const highscoresTable = document.querySelector(".grid-container");
+      htmlRows.forEach((row) => {
```
Comment (elnino97) [inline public/javascripts/account-main.js:26]: Why not append html to the document while iterating in the loops above? Also consider switching to ES6 methods, looping is not ideal

File: views/account/account-main.ejs
Comment (DmitrijSidorkin) [reply views/account/account-main.ejs:1]: Its my first time using a grid properly so I am not too sure, but this empty div seems necessary to me. It seems necessary to keep the grid structure intact. 

The green cell represents the empty div 
image

File: public/javascripts/account-main.js (hunk 1 above)
Comment (DmitrijSidorkin) [reply public/javascripts/account-main.js:26]: Wouldnt appending constantly while looping be more costly performance-wise? Changed a bit how the `div`s are generated, so can add them all in one go now, which seems more efficient.

Comment (elnino97) [reply public/javascripts/account-main.js:26]: That is a good point actually, yes you should keep it like that then.

File: views/account/account-main.ejs
Comment (elnino97) [reply views/account/account-main.ejs:1]: No worries, for a first timer it's pretty decent. You could work around, but for now let's keep it this way.

**Review by elnino97 — APPROVED** [review]

**Pull request merged**